crates/core/src/spk_client.rs

```diff
@@ -220,35 +217,45 @@ pub struct SyncRequest<I = ()> {
     inspect: Box<InspectSync<I>>,
 }
 
-impl<I> Default for SyncRequest<I> {
-    fn default() -> Self {
-        Self {
-            chain_tip: None,
-            spks: VecDeque::new(),
-            spks_consumed: 0,
-            txids: VecDeque::new(),
-            txids_consumed: 0,
-            outpoints: VecDeque::new(),
-            outpoints_consumed: 0,
-            inspect: Box::new(|_, _| {}),
-        }
-    }
-}
-
 impl<I> From<SyncRequestBuilder<I>> for SyncRequest<I> {
     fn from(builder: SyncRequestBuilder<I>) -> Self {
         builder.inner
     }
 }
 
 impl<I> SyncRequest<I> {
-    /// Start building a [`SyncRequest`].
-    pub fn builder() -> SyncRequestBuilder<I> {
+    /// Start building [`SyncRequest`] with a given `start_time`.
+    pub fn builder(start_time: u64) -> SyncRequestBuilder<I> {
```

Comment:
We need to explain what's the purpose of `start_time` here.

Reply:
Added.